.editorconfig

```diff
@@ -0,0 +1,13 @@
+# editorconfig.org
```

Comment:
shouldn't that be in gitignore?

Reply:
Are you talking about the file itself?

Reply:
yes! Though idk what it's supposed to be, seemed like a vscode setting file from what I found